csrc/ops/alias.cpp

```diff
@@ -157,19 +167,23 @@ TensorView* reshape(TensorView* inp_tv, const std::vector<Val*>& new_sizes) {
             .iter_type(IterType::Symbolic)
             .is_rfactor_domain(true)
             .build();
-    rfactor_domain.at(i) = rf_id;
+    view_rfactor_domain.push_back(rf_id);
   }
 
   auto out_tv = IrBuilder::create<TensorView>(
       IrBuilder::create<TensorDomain>(
           root_domain,
-          rfactor_domain,
-          rfactor_domain,
-          TensorDomain::getContiguityFilledWith(rfactor_domain, true)),
+          view_rfactor_domain,
+          view_rfactor_domain,
+          TensorDomain::getContiguityFilledWith(view_rfactor_domain, true)),
       inp_tv->dtype());
 
   IrBuilder::create<ViewOp>(inp_tv->container(), out_tv, inp_tv);
 
+  if (found_static_broadcast) {
```

Comment:
@jacobhinkle I think I did the right thing here now.

Reply:
Yeah I like this.